packages/cli-repl/test/repl-helpers.ts

```diff
@@ -17,11 +17,15 @@ chai.use(chaiAsPromised);
 // to wait for these to finish.
 const tick = promisify(setImmediate);
 
+// We keep an additional index as we might create two temp directories
```

Comment:
this one confuses me :)

Reply:
It was... when I created git repos locally in tests I got conflicts as `new Date()` was not sufficient to create a unique folder name. I've left it in as it would prevent the same issue in the future.
I don't need it right now anymore so I can remove if you say we should strip.

Reply:
Fwiw, we do use `Math.random()` in a few other places as well. Doing that here would sound good to me in terms of consistency :+1:

Reply:
(Or, to be clear, leaving it in this PR is also just fine with me!)